api/src/pcapi/core/search/backends/algolia.py

```diff
@@ -418,6 +419,10 @@ def serialize_offer(cls, offer: offers_models.Offer, last_30_days_bookings: int)
             except AttributeError:
                 macro_section = None
 
+        product_extra_data = offer.product.extraData
+        gtl_code = product_extra_data.get("gtl_id") if product_extra_data else None
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❔ question:** ‏`product_extra_data` peut être None ?
Si c'est juste pour gérer le cas `{}` pas besoin de `if product_extra_data else None`

Reply:
https://github.com/pass-culture/pass-culture-main/blob/0f1bd5d5f4732a4cf8317c700c21fea6ddb4140f/api/src/pcapi/core/offers/models.py#L116

ça peux être None